README.md

```diff
@@ -1,8 +1,8 @@
-# Container-agnostic service providers
+# Standard service providers
 
-**Work in progress.**
+This project tries to find a solution for cross-framework modules (aka bundles) through **standard container configuration**. It is part of the [container-interop](https://github.com/container-interop/container-interop) group.
```

Comment:
You'll noticed I've changed "container-agnostic" with "standard". I think it's simpler to understand what it's all about, and it's more direct. After all we are working on a PSR/standard so…